src/db/TickSchema.ts

```diff
@@ -16,8 +16,8 @@ export const TickSchema = new Schema<TickType>({
   notes: { type: Schema.Types.String, required: false },
   climbId: { type: Schema.Types.String, required: true, index: true },
   userId: { type: Schema.Types.String, required: true, index: true },
-  style: { type: Schema.Types.String, required: true, default: '' },
-  attemptType: { type: Schema.Types.String, required: true, index: true, default: '' },
+  style: { type: Schema.Types.String, enum: ['Lead', 'Solo', 'TR', 'Follow'], required: false },
+  attemptType: { type: Schema.Types.String, enum: ['Onsight', 'Flash', 'Redpoint', 'Pinkpoint', 'Frenchfree', 'Fell/Hung', 'Send', 'Attempt'], required: false, index: true },
```

Comment:
I'm not the biggest fan of the field `attemptType`, having a possible value of `Attempt`. Perhaps something like `Fell` (for boulders), instead? We use `Fell/Hung` for routes, so I feel like `Fell` makes sense for boulders.

Reply:
Also, this is perhaps an uncommon use case, but are there enough people who would want an `Aid` option? I like having `FrenchFree` as an option, and `Aid` can certainly be considered distinct from FrenchFree.

Reply:
I'm not a boulderer so idk if fell or attempt makes more sense here. I would think some people might like attempt as opposed to fell if they got on a project just to work a particular move or sequence, but I'll defer to people who boulder more than I do.

Reply:
> but are there enough people who would want an Aid option
I could go either way on this. The route grade is already marked with an aid rating. Again I don't aid climb ever and I don't know if I would ever use "french free", but I am open to including it or not including it

Reply:
I'm certainly not opposed to improving on what MP does, but I appreciate the simplicity of MP as opposed to something like 27 crags where theres like 20 options. I also know the majority of users probably will also be users of MP right now. I would lean towards mirroring MP for now, but I don't have a hard stance either way